src/server.h

```diff
@@ -1797,8 +1800,10 @@ struct redisCommand {
     char *sflags;   /* Flags as string representation, one char per flag. */
     keySpec key_specs_static[STATIC_KEY_SPECS_NUM];
     /* Use a function to determine keys arguments in a command line.
-     * Used for Redis Cluster redirect. */
+     * Used for Redis Cluster redirect (may be NULL) */
     redisGetKeysProc *getkeys_proc;
+    /* Array of subcommands (may be NULL) */
+    struct redisCommand *subcommands;
```

Comment:
Is this used?

Reply:
yes - it is used to populate `subcommands_dict` (much like `sflags` is used to populate `flags`)